scripts/release-notes.py

```diff
@@ -97,7 +97,7 @@ def main(token, repo, branch, num_commits, exclude_dependabot, verbose):
 
     progress_iterator = 0
     for commit in commits:
-        if verbose:
+        if repo == 'jaeger':
```

Comment:
this is rather odd - why do you only want progress reporting for one specific repo?

Reply:
It's equivalent to what we had before. In jaeger Makefile, we had `--verbose` set for `changelog` target, then it's disabled for the `jaeger-ui` repo to prevent progress from showing for the jaeger-ui section.

The reason for doing this is because we shouldn't need to use `--verbose` flag by default and it should be respected for both `jaeger` and `jaeger-ui` repos. Either way, it's not ideal, but I think it's better if `--verbose` was respective irrespective of the repo.

Reply:
Hmmm... actually, it's a good callout. I just realised we're using the same script for jaeger-ui. I'll revert this change in that case.